### DISCUSSION TITLE
URGENT- Moved projects, deleted my team, domains have been lost?

### DISCUSSION THREAD

**Discussion by umutahmet** [vercel/community#3699] [Account Issue]
### Question

Really embarrassing but I'm scaling back from my team and thought by moving my projects I would have taken the domains with me to my hobby account? Three of my websites are down. Need help.

**Comment by amyegan** [comment]
Hey @umutahmet. If you let me know the domains, I can look for where they are stuck 🙇

**Reply by umutahmet** [reply]
Hey @amyegan, thanks for your reply. 

I got to the bottom of it after my initial panic. I was moving too quickly to notice messaging in the dashboard before moving/deleting. 

However, I do have feedback on your guides for moving projects here: [https://vercel.com/docs/projects/overview#transferring-a-project](https://vercel.com/docs/projects/overview#transferring-a-project)
The table suggests the domain will be transferred with the project and that's not what happened. Instead, (from my understanding) I need to move the domain first, then move the project. 

Lastly, once I deleted my PRO team, I was unable to raise a sev 1 to support because my team had been deleted. This might be an edge case but I believe I should still be able to get in touch as I was still within my payment cycle.

Anyway, just my thoughts. I've already given this feedback to your support but thought it was worth mentioning it to you too :) Thanks
